libtesting-ui/src/main/java/com/mapbox/navigation/testing/ui/utils/coroutines/Adapters.kt

```diff
@@ -113,6 +115,49 @@ fun MapboxNavigation.rawLocationUpdates(): Flow<Location> {
     )
 }
 
+sealed interface NavigationRouteAlternativesResult {
```

Comment:
nit

It seems that this can be replaced with `Expected<RouteAlternativesError, OnRouteAlternatives>`. I'd prefer this way as a common. Also, this would be more flexible as we broadly use `Expected` across the code.

Reply:
It's a matter of taste probably. I don't like using Expected as is isn't as integrated with kotlin compiler as sealed classes/interfaces and I like using smart casts 🙂